sns/cm/repair/ut/cp_common.c

```diff
@@ -147,7 +147,8 @@ void cp_prepare(struct m0_cm_cp *cp, struct m0_net_buffer *buf,
 	cp->c_ag->cag_cm = cm;
 	if (!is_acc_cp)
 		cp->c_ops = &m0_sns_cm_repair_cp_ops;
-	cp->c_ops = &m0_sns_cm_acc_cp_ops;
```

Comment:
you can move this line above the 'if' condition.

Reply:
If I move it above 'if' condition, then it will again generate same codacy issue.
In both scenario, if condition is getting evaluated.
So with current change cp->c_ops, will be evaluated once, if I move it above, it will be overwritten if if(!is_ac_up) evaluates to true.

Reply:
As per my understanding, the present code is overwriting cp->c_ops unconditionally and which is why Codacy is generating the error. 
Did you check that Codacy still generates the same issue when the assignment is moved above the 'if' ?